ipm_library/package.xml

```diff
@@ -10,12 +10,8 @@
   <depend>geometry_msgs</depend>
   <depend>ipm_interfaces</depend>
   <depend>python3-numpy</depend>
-  <depend>sensor_msgs</depend>
```

Comment:
`sensor_msgs`, `std_msgs`, `vision_msgs` and `shape_msgs` are still used in the `ipm.py`

Reply:
I think in practice, it doesn't matter if these dependencies are listed or not, because they are all dependencies of ``ipm_interfaces``. I'm not sure what's considered better though, to list them explicitly, or not to.

Reply:
I'm actually geneuinely interested, so I've posted a question on ROS Answers:
https://answers.ros.org/question/405832/listing-dependencies-in-packagexml-that-are-also-dependency-of-dependency/

Reply:
If for some reason, `ipm_interfaces` would not depend on `sensor_msgs` anymore, we would get missing dependencies. Yes, probably in that case, we would get more problems...

Reply:
+ one for the thing @jaagut says. Changes in dependencies of upstream dependencies should not affect the direct dependencies of our code.

Reply:
The response I got was exactly what you said @jaagut - 
https://answers.ros.org/question/405832/listing-dependencies-in-packagexml-that-are-also-dependency-of-dependency/

I'm going to add them back in. It is unfortunate that the CI works either way. I wish if the CI indicated a missing dependency if not explicitly stated :thinking: